src/render/gpu/SDL_render_gpu.c

```diff
@@ -61,6 +62,8 @@ typedef struct GPU_RenderData
         SDL_GPUTransferBuffer *transfer_buf;
         SDL_GPUBuffer *buffer;
         Uint32 buffer_size;
+
+        SDL_GPUBuffer *present_quad_buffer;
```

Comment:
This buffer will contain a single fullscreen quad for presenting to a GPU texture. I am not sure if it would be possible to store in general vertices buffer, as it can be overwritten?